app/views/includes/foot.jade

```diff
@@ -10,20 +10,24 @@ script(type='text/javascript', src='/lib/underscore/underscore-min.js')
 script(type='text/javascript', src='/lib/bootstrap/dist/js/bootstrap.js')
 
 //AngularJS
+
```

Comment:
But from the PR I can see that your push added these changes. If the spaces are unnecessary, get rid of them then and make your PR cleaner 😝😉